src/modules/qt/filter_gpsgraphic.yml

```diff
@@ -0,0 +1,469 @@
+schema_version: 7.0
+type: filter
+identifier: gpsgraphic
+title: GPS Graphic
+version: 1
+copyright: Meltytech, LLC
+creator: Daniel F
+license: LGPLv2.1
+language: en
+tags:
+  - Video
+description: Overlay GPS-related graphics onto the video
+
+parameters:
+  - identifier: gps.file
```

Comment:
Congratulations, it compiles fine on Qt 6 now!
Please change "gps.file" in both filters to "resource" as that is our standard property name for this kind of parameter, and the xml module and Shotcut already know how to handle this property name to make and handle a relative path. Since I did not catch this for gpstext already you should add it as an undocumented backup (not published in this YAML) for when "resource" returns NULL. Also, you should consider making this the filter's argument (`arg` in `filter_gpsgraphic_init()`). Finally, it is not indicated as required, but it seems like it is to be useful. Making it required and argument does not necessarily mean it needs to be supplied as an argument. Also, you do not need to enforce the requirement, but it just helps logically to understand. Basically, if a user tries to use the filter and does not set this parameter and cannot get it to do anything, `required: yes` is a strong hint that you need to supply it to have some effect.

Reply:
> Since I did not catch this for gpstext already you should add it as an undocumented backup (not published in this YAML) for when "resource" returns NULL

Just to double check I understood this: you mean replace all references of gps.file with resource (including in .yml for gps_text) but keep it backwards compatible in the .cpp to also accept gps.file? 

> Also, you should consider making this the filter's argument (arg in filter_gpsgraphic_init()). 

I'm not sure how that arg is populated, do I need to do more than this line?
`	mlt_properties_set_string( properties, "resource", arg );`

Reply:
> replace all references of gps.file with resource

Yes

> keep it backwards compatible in the .cpp to also accept gps.file?

Only for gpstext since that is the only one released so far. We do not want to break existing projects and scripts.

> do I need to do more than this line?

No, that works good even when NULL.